app/src/main/java/com/sun/ui/base/BaseViewHolder.kt

```diff
@@ -3,5 +3,5 @@ package com.sun.ui.base
 import androidx.databinding.ViewDataBinding
 import androidx.recyclerview.widget.RecyclerView
 
-class BaseViewHolder<out T : ViewDataBinding> constructor(val binding: T) :
+open class BaseViewHolder<out T : ViewDataBinding> constructor(val binding: T) :
```

Comment:
về mặt OOP, nếu không có đối tượng nào cụ thể tên là BaseViewHolder mà chỉ là con của nó thì để là abstract (trừu tượng ) 
```suggestion
abstract class BaseViewHolder<out T : ViewDataBinding>(val binding: T) :
```

Reply:
chỗ này vì đang return trực tiếp BaseViewHolder ở trong BasePagingAdapter nên không để nó abstract được :<